_data/simple-icons.json

```
@@ -1912,6 +1912,12 @@
             "source": "https://www.backblaze.com/company/news.html",
             "guidelines": "https://www.backblaze.com/company/news.html"
         },
+        {
+            "title": "Backbone",
+            "hex": "000000",
+            "source": "https://playbackbone.com/press",
+            "guidelines": "https://playbackbone.com/press"
```

Now is `backbone.com`

```suggestion
            "source": "https://backbone.com/press",
            "guidelines": "https://backbone.com/press"
```